specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/storagecache.json

```diff
@@ -1606,7 +1606,7 @@
           "type": "string"
         },
         "identity": {
-          "$ref": "#/definitions/CacheIdentity",
+          "$ref": "#/definitions/IdentityProperties",
           "description": "The identity of the cache, if configured."
```

Comment:
"cache" in description

Reply:
I'm seeing this in the Cache object.  So the description of this field still works.